src/components/Observations/index.tsx

```diff
@@ -71,14 +86,14 @@ export default function Observations(): JSX.Element {
         <ObservationPlantingZoneDetails />
       </Route>
       <Route exact path={APP_PATHS.OBSERVATION_DETAILS}>
-        <ObservationDetails />
+        <ObservationDetails {...searchInputProps} />
       </Route>
       <Route exact path={APP_PATHS.PLANTING_SITE_OBSERVATIONS}>
-        <ObservationsHome />
+        <ObservationsHome {...searchInputProps} />
       </Route>
       <Route path={'*'}>
-        <ObservationsHome />
+        <ObservationsHome {...searchInputProps} />
```

Comment:
search input props are lifted in parent and shared across observations home and details page (both search for zones)